### ISSUE TITLE
Request: Service name(s) in the process column list (2)

### ISSUE THREAD

**Issue by AltF5** [winsiderss/systeminformer#1254]
### Description of the feature, modification, idea or suggestion

As per here https://github.com/winsiderss/systeminformer/issues/406

Are you open to pull requests to implement this ? I’m sure things are busy with the MS rebranding effort, etc

**Comment by jxy-s** [comment]
Looks like @dmex already said you should PR what you have in the other thread. I'm okay with this addition as along as there is not a negative performance impact. Feel free to open a PR.

**Comment by AltF5** [comment]
Here's my Pull Request -- Please modify as y'all see fit, but I've been testing this out for a while and it works perfectly all the way back since June 2019 to now in June 2022 with SystemInformer

https://github.com/winsiderss/systeminformer/pull/1272
![PH SI 3](https://user-images.githubusercontent.com/1288507/174016652-df8051cf-63f9-4138-a731-9db41cce1881.png)

**Comment by Biswa96** [comment]
I think 'Services' tab also has the same features/columns.

**Comment by AltF5** [comment]
Not quite, IMHO respectfully :). As per dicussion here https://github.com/winsiderss/systeminformer/issues/406 from 2019 --> 2021 I (and some other users) definitely see the advantage of having the services listed "in-line" within the treeview, not just to appear by tooltip.

dmex does call out the overkill number of ways to view service info :), but I feel having services in a flat list, especially for multiple svchosts is of great use, especially if you are "use to seeing the tree" in a certain order and can learn which svchosts, etc typically load / appear in the tree (are they started earlier, delayed, restarted, etc).

Also, in the services tab there is no 1-to-many relationship (for 1 process, what's the list of services it's hosting), which is where I feel this column shines (basically taking the tooltip and placing it into a column).

If dmex, jxy, or AlexIon really have heartburn with this, then perhaps the other columns could be included. I hate to always have to make these changes locally whenever I download a new release, since I find them invaluable, but if at least the count of Groups & Privileges (and even the Service Count perhaps) were in there, then that would be a bit less work. The Group and Privilege count is immensely helpful when working with tokens, impersonation, and searching for viable candidates... I'd suspect @diversenok would agree :D

**Comment by 0x4E69676874466F78** [comment]
I wanted this functionality for maximum convenience.
